### PR TITLE
Fix build warnings. Update Attic_ASTM_C1340_Base_Case.idf file.

### PR DESCRIPTION
**Pull request by mitchute**
Pull request overview
---------------------
Cleanup for #6021 and #6027.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [x] Code style (parentheses padding, variable names)
 - [x] Functional code review (it has to work!)
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check -- verify this
 - [x] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by mitchute** [comment]
@mjwitte FYI

File: testfiles/Attic_ASTM_C1340_Base_Case.idf [hunk 1]
```diff
@@ -126,123 +151,194 @@
     Yes,                     !- Use Weather File Rain Indicators
     Yes;                     !- Use Weather File Snow Indicators
 
-!-   ===========  ALL OBJECTS IN CLASS: GROUNDTEMPERATURES ===========
 
-  Site:GroundTemperature:BuildingSurface,18.3,18.2,18.3,18.4,20.1,22.0,22.3,22.5,22.5,20.7,18.9,18.5;
+!-   ===========  ALL OBJECTS IN CLASS: SITE:GROUNDTEMPERATURE:BUILDINGSURFACE ===========
+
+Site:GroundTemperature:BuildingSurface,
+    23.0,                    !- January Ground Temperature {C}
```
Comment (mjwitte) [inline testfiles/Attic_ASTM_C1340_Base_Case.idf:158]: @mitchute Did you intend to change the ground temps here?  I guess looking further down this is more than just a transition/cleanup.

Comment (mitchute) [reply testfiles/Attic_ASTM_C1340_Base_Case.idf:158]: @mjwitte Yes, I wish I had caught this earlier. This is the latest file we have been using to compare against AtticSim, which I'm assuming we will want to be kept here.

**Comment by mjwitte** [comment]
@mitchute OK, so, it didn't run here, because this branch doesn't have the fixed cmake list file (it's behind develop slightly).  I ran it locally and it runs to completion.  But this file needs the standard comment block at the top so it will index properly into the examplefile.html directories.

**Comment by mjwitte** [comment]
Merging this now.  Can add the comment block later.

**Pull request merged**